lessons/codap-exploring/langs/en-us/index.adoc

```diff
@@ -122,7 +116,9 @@ Brainstorm other scenarios where Booleans are useful in and out of the programmi
 === Investigate
 [.lesson-instruction]
 --
-In pairs, complete @printable-exercise{pages/booleans.adoc}, making predictions about what a variety of Boolean expressions will return and testing them in the editor.
+- Turn to @printable-exercise{pages/booleans.adoc}.
+- Make a prediction about what a variety of Boolean expressions will return, and then test them in the @starter-file{boolean}.
```

Comment:
Honestly I'd suggest hitting kids over the head with it: "Write down your predictions _before_ testing them in CODAP". I know it's in the worksheet already, but in practice kids need to be reminded like crazy

Reply:
Good idea.